### ISSUE TITLE
EntityDamageByEntityEvent setBaseDamage Bug

### ISSUE THREAD

**Issue by DANRULEZ123** [Benedikt05/BetterAltay#274]
### Issue description

- Expected result: The base damage is changed to the set amount with attack speed normal.
- Actual result: Calling `$event->setBaseDamage(1.0); // or 1 or any int/float` completely removes attack speed.

### Steps to reproduce the issue
1. Make an Event in your plugin.
2. Make the function for your event (EntityDamageByEntityEvent) and call setBaseDamage();

### OS and versions

* Altay: 1.21.1
* PHP: 

- If you remove all plugins, does the issue still occur?
 - No

- If the issue is **not** reproducible without plugins:
  - Can you provide sample, *minimal* reproducing code for the issue? If so, paste it in the bottom section

### Crashdump, backtrace or other files

public function onDamage(EntityDamageByEntityEvent $event):void{
$damager = $event->getDamager();
$entity = $event->getEntity();
If($event->isCancelled()){
return;
}
if($damager instanceof Player and $entity instanceof Player){
$event->setBaseDamage(1); // Completely removes attack speed...
$damager->sendMessage("Damage Event Triggered.");
}
}

**Comment by vp817** [comment]
I don't think this is the right function for this thing. Since you want to make the speed of attack faster, consider using "setAttackCooldown".
